docs/user/ml/index.asciidoc

```diff
@@ -187,14 +190,20 @@ image::user/ml/images/ml-change-point-detection.png[Change point detection UI]
 
 Select a function and a metric field, then pick a date range to start detecting 
 change points in the defined range. Optionally, you can split the data by a 
-field. If the cardinality of the split field is greater than 10,000, then only 
-the first 10,000, sorted by document count, are analyzed.
-
-If a change point is detected, a chart visualizes where the change point was identified in
-the time window analyzed, making the interpretation easier. If you split the analysis by a 
-field, a separate chart is displayed for every partition with a detected change 
-point. You can view the type of change point in the chart as well as its value 
-and the time when the change happened. The corresponding `p-value` indicates how 
-extreme the change is; lower values mark more significant changes. You can use 
-the change point type selector to filter the results by specific types of change 
-points.
\ No newline at end of file
+field. If the cardinality of the split field exceeds 10,000, then only the first 
+10,000, sorted by document count, are analyzed. You can select multiple metric 
+fields and split fields to identify change points.
+
+When a change point is detected, a row displays basic information including the 
+time, a preview chart, the type of change point, p-value, field name, and value. 
```

Comment:
shall we clarify the `time` here? e.g. `timestamp of the bucket where a change point has been detected`

Reply:
and the `field name` - `name of the split field`,
`value`- a value of the split field or name of the partition 🤔